core/gather/driver/wait-for-condition.js

```diff
@@ -540,7 +540,8 @@ function waitForUserToContinue(driver) {
   }
   /* c8 ignore stop */
 
-  driver.defaultSession.setNextProtocolTimeout(2 ** 31 - 1);
+  // Do "- 51" instead of "- 1" because we always add 50ms for the Puppeteer timeout
+  driver.defaultSession.setNextProtocolTimeout(2 ** 31 - 51);
```

Comment:
Sorry jinx on the PR update.

This one actually makes sense to be the max int32 since the users manually decides when to continue. I can imagine plenty of scenarios where I want to inspect the page for longer than 5 min.